### PR TITLE
Add responsive to main page

### PR DESCRIPTION
**Pull request by yummta**

### CODE REVIEW

File: public/assets/css/base/helpers.css [hunk 1]
```diff
@@ -18,4 +22,13 @@
...
   padding: 48px
+}
+
+@media screen and (max-width:480px) {
+  .padding48{
+    padding: 24px;
+  }
+  .padding24{
+    padding: 16px;
+  }
```
Comment (liannivin) [inline public/assets/css/base/helpers.css]: Good work. I would say to take care naming your classes.

File: public/assets/css/base/helpers.css [hunk 2]
```diff
@@ -18,4 +22,13 @@
 }
 .padding48{
   padding: 48px
+}
+
+@media screen and (max-width:480px) {
+  .padding48{
```
Comment (cdsandoval) [inline public/assets/css/base/helpers.css]: This name of classes are awesome, it makes sense. Good job !!

File: public/assets/css/base/helpers.css [hunk 3]
```diff
@@ -18,4 +22,13 @@
...
+    padding: 24px;
+  }
+  .padding24{
```
Comment (cdsandoval) [inline public/assets/css/base/helpers.css]: This one break my heart, name of class says 24 and then padding: 16 px. 🥇

File: public/assets/images/delete.svg [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (cdsandoval) [inline public/assets/images/delete.svg]: I like to save it in a individual file. Are you calling this file to the HTML ?

File: public/main.html [hunk 5]
```diff
@@ -35,10 +34,31 @@
...
+          </svg>
+        </div>
         <h1 class="title">IRC - Welcome @</h1>
       </header>
-      <sections id="channel-list" class="channels padding16">
+      <sections id="channel-list" class="channels padding16 relative">
         <div class="list-channels">
+          <div id="js-close-channels" class="button-icon close">
+            <svg width="16" height="16" viewBox="0 0 348.333 348.334">
+              <path d="M336.559,68.611L231.016,174.165l105.543,105.549c15.699,15.705,15.699,41.145,0,56.85
```
Comment (cdsandoval) [inline public/main.html]: If you're saving a svg file, wouldn't be better to call it within an image element? Overall, well job doing it.

**Pull request merged**